baybe/recommenders/pure/base.py

```diff
@@ -39,6 +55,9 @@ def recommend(  # noqa: D102
         train_y: Optional[pd.DataFrame] = None,
     ) -> pd.DataFrame:
         # See base class
+        if self.user_callable is not None:
```

Comment:
it should be a list instead of a single callable

Reply:
Didn't we agree on having just a single Callable for this very basic example? And also, since this will change anyway in the near future, is it necessary to still adjust this?

Reply:
did we really agree on this? the difference is 2 lines

Reply:
I'm fine with both, so let's make it a list.

Reply:
Yes, let's to a list. And while we are currently on it: shall we keep `user_callable(s)`? Or have we ever talked about the attribute name? Alternative could be simply "hooks", or similar. But I have no strong opinion here

Reply:
Also no strong opinion, but since this will be made redundant later I would just keep this as-is

Reply:
alright, changed it to list only